composer.json

```diff
@@ -44,6 +43,10 @@
     "monolog/monolog": "*"
```

Comment:
I think adding `"packfire/php5.3-compat": "*"` to `require-dev` will get the tests to pass and will still satisfy the desire not to install it for everyone.